src/rsz/src/Rebuffer.cc

```diff
@@ -136,6 +131,77 @@ void pruneCapVsAreaOptions(StaState* sta, BufferedNetSeq& options)
   options.resize(si);
 }
 
+static const RiseFallBoth* transitionGcd(const RiseFallBoth* a,
```

Comment:
What is 'Gcd'?

Reply:
It's the most specific transition that subsumes both `a` and `b`, (where `a`,`b` can be rise/fall/both/null). It's named by analogue to greatest common divisor. Should I rename? I've added an explanatory comment in any case

Reply:
If we are going to define an abstract algebra on transitions then this seems like the least upper bound or supremum.  Less mathematical I might use mergeTransitions.